helix-core/src/main/java/org/apache/helix/manager/zk/ZkBucketDataAccessor.java

```diff
@@ -206,7 +213,7 @@ public <T extends HelixProperty> boolean compressedBucketWrite(String rootPath,
     buckets.add(binaryMetadata);
 
     // Do an async set to ZK
-    boolean[] success = _zkBaseDataAccessor.setChildren(paths, buckets, AccessOption.PERSISTENT);
+    boolean[] success = _zkBaseDataAccessor.setChildren(paths, buckets, AccessOption.PERSISTENT, _znodeTTLms);
```

Comment:
I think if we using ttl, accessOption also need to be updated to use PERSISTENT_WITH_TTL